tests/js/spec/components/modals/dashboardWidgetLibraryModal.spec.jsx

```diff
@@ -117,4 +119,20 @@ describe('Modals -> DashboardWidgetLibraryModal', function () {
 
     container.unmount();
   });
+
+  it('disables save button if widget limit is exceeded', function () {
+    // Checking initial modal states
+    const mockApply = jest.fn();
+    const closeModal = jest.fn();
+    types.MAX_WIDGETS = 1;
+    const container = mountModal({initialData}, mockApply, closeModal);
+
+    const selectButtons = screen.getAllByRole('button');
```

Comment:
Is it possible to get more specific than this with `getAllByRole`? I think it takes a `name` arg, might be able to do `screen.getAllByRole('button', {name: "Select"})` so it could be a bit more robust in case future buttons appear

Reply:
I got rid of all this in a previous PR with all the design changes!

Reply:
I saw! The easy to understand selectors are a blessing.